dsgrid/registry/project_registry_manager.py

```diff
@@ -355,6 +396,145 @@ def _register_dimensions_from_models(
         dimensions.clear()
         return dimension_references
 
+    def _register_dimensions_from_subset_dimensions(
```

Comment:
Isn't this just `_register_subset_dimensions`?

Reply:
Sure